### DISCUSSION TITLE
Pnpm deploy command deploy nothing

### DISCUSSION THREAD

**Discussion by MathieuAndrade** [pnpm/pnpm#7619] [Q&A]
Hello ! 

I try to use `pnpm deploy` as mentioned in [documentation](https://pnpm.io/cli/deploy).
This command seem to "deploy" a package from workspace with his node_modules in seperated target folder.
The documentation explicit mentionned **The target directory will contain a portable package that can be copied to a server and executed without additional steps.**

But unfortunately, the command only copy the content of targeted package and create an empty node_modules folder with a `pnpm-lock` file. 😞 
So a tried to copy the content of the target folder (after run deploy command) and try to run a `pnpm install` (maybe ?, for the test), but the others packages of workspace mentioned in lock file failed to install. 

```js
 WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@pnpm-test/logger@workspace:^" is in the dependencies but no package named "@pnpm-test/logger" is present in the workspace
```

I don't understand why this isn't working and what if I missed something in the documentation ?

Here is a minimal repo for reproduction: [Pnpm deploy test](https://github.com/MathieuAndrade/pnpm-deploy-test) and [Codespace link maybe](https://codespaces.new/MathieuAndrade/pnpm-deploy-test?quickstart=1)

I use Pnpm 8.15.0 and Windows 10 (with WSL 2)

Thanks for your help 🙏

**Comment by aDarcy99** [comment]
@MathieuAndrade Did you end up getting this working?
